01-Git_Bash_Make/Task2_bash_script/script

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
While it isn't strictly required, it's preferable to add `.sh` extension to shell scripts.

Reply:
Here and later commit messages seem to be waiting for some cleanup.